tests/pass-dep/libc/libc-affinity.rs

```diff
@@ -89,18 +89,37 @@ fn large_set() {
     assert_eq!(err, 0);
 }
 
+fn get_small_cpu_mask() {
+    let mut cpuset: cpu_set_t = unsafe { core::mem::MaybeUninit::zeroed().assume_init() };
+
+    // should be 4 on 32-bit systems and 8 otherwise for systems that implement sched_getaffinity
+    let step = size_of::<std::ffi::c_ulong>();
+
+    // currently the maximum number of CPUs that miri supports, and also the number that a
+    // libc::cpu_set_t can hold.
+    let max_cpus = 1024;
+
+    for i in (0..=max_cpus / step).step_by(step) {
```

Comment:
I don't think we have to exhaustively test this all the way up to 1024. `for i in `(0..=2).map(|x| x*step)` should be enough.